### DISCUSSION TITLE
Quick way of upgrading packages in a separate monrepo

### DISCUSSION THREAD

**Discussion by jeswr** [comunica/comunica#990] [Q&A] [answered]
Most of the packages in https://github.com/comunica/comunica-feature-reasoning/ are still depending on v2.0.0 packages of comunica - is there an existing way to quickly upgrade all the packages in the reasoning repo to use v2.2.0 packages from the core?

**Comment by rubensworks** [comment, marked as answer]
I just do it with a search/replace-all myself, but that can be error-prone, so I would love to know about a better way as well if there is one 😄

**Reply by jeswr** [reply]
Oh almost forgot - this little script works like in a charm for updating in the reasoning repo (https://github.com/comunica/comunica-feature-reasoning/blob/6a4224177c39e012d201e88312d9de632d4f7631/lerna.js#L108-L125). I'll create an issue to port it over here.
